client/blocks/inline-help/inline-help-contact-view.jsx

```diff
@@ -28,11 +29,21 @@ const InlineHelpContactView = ( {
 		return <PlaceholderLines />;
 	}
 
-	if ( supportVariation === SUPPORT_FORUM ) {
-		return <InlineHelpForumView />;
-	}
-
-	return <HelpContact compact selectedSite={ selectedSite } />;
+	return (
+		<React.Fragment>
```

Comment:
Fun fact!

If we want to we can do this instead…

```js
import React, { Component, Fragment } from 'react';

…

<Fragment>
	…
</Fragment>
```

Reply:
Thanks @dmsnell :)
I've usually err'ed on the side of dot-ing it to avoid confusion (what the heck's a `Fragment`?!), but I think folk are probably a bit more familiar with it these days!

I'll destructure it before merging - I do generally prefer it this way myself too :)

Reply:
sure! I haven't heard any confusion over it and the import's at the top. 🤷‍♂️